apps/volunteer/src/pages/volunteers/_components/VolunteerRecruitItem.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  AspectRatio,
+  Box,
+  HStack,
+  Image,
+  Text,
+  VStack,
+} from '@chakra-ui/react';
+import { MouseEvent } from 'react';
+import ApplicantStatus from 'shared/components/ApplicantStatus';
+import InfoSubText from 'shared/components/InfoSubtext';
+import LabelText from 'shared/components/LabelText';
+
+type Recruitment = {
+  id: number;
+  title: string;
+  shelterName: string;
+  shelterProfileImage: string;
+  volunteerDate: string;
+  volunteerDateDday: number;
+  applicantCount: number;
+  recruitmentCapacity: number;
+};
+
+type VolunteerRecruitItemProps = {
+  recruitment: Recruitment;
+  onClickItem: (event: MouseEvent<HTMLDivElement>) => void;
+};
+
+export default function VolunteerRecruitItem({
+  recruitment,
+  onClickItem,
+}: VolunteerRecruitItemProps) {
+  const {
+    id,
+    title,
+    shelterName,
+    shelterProfileImage,
+    volunteerDate,
+    volunteerDateDday,
+    applicantCount,
+    recruitmentCapacity,
+  } = recruitment;
+
+  return (
+    <Box
+      data-id={id}
+      p={4}
+      borderBottom="1px"
+      borderColor="gray.200"
+      onClick={onClickItem}
+    >
+      <HStack spacing={3} justifyContent="center">
```

Comment:
```suggestion
      <HStack spacing={3} justify="center">
```

이렇게 단축할 수 있을 것 같습니다!!

Reply:
디자인을 다시 보니까 텍스트 끼리의 gap 도 제대로 설정되어있지 않아서 css 수정을 다시하고 재리뷰 요청 드리겠습니다!